lib/assets/javascripts/_modules/search.js

```diff
@@ -196,6 +196,7 @@
         .attr('aria-hidden', 'false')
       $html.addClass('has-search-results-open')
       $tocNav.addClass('search-results-open')
+      $searchHelp.attr('hidden', 'true')
```

Comment:
What happens if the user clears the search input field? Presumably the help text stays `aria-hidden`, when it should actually be available to assistive tech again?

Reply:
If they clear the search input nothing will happen, as we've removed the typeahead, so I think this is fine?